Path.json

```
@@ -563,6 +563,17 @@
 				}
 			]
 		},
+		{
+					"group": 2
+					"maxspeed": 120
+					"start": "KD"
+					"end": "KSO"
+					"twistingfactor": 0.2
+					"length": 19
+					"electrified": true
+				}
+			]
+		},
```

Da fehlt ein Komma hinter der 2, und nicht nur da.

Welchen Editor benutzt du  eigentlich? In VSCode sieht das bei mir so aus - und beim Hovern mit dem Mauszeiger über die Kringel, sagt er auch, was ihm nicht paßt.

`twistingFactor` - mit großem F, `maxSpeed` mit großem S (case sensitive). und dann hattest du noch eine geschweifete und eine Eckige Klammer zu viel.

 (Warum? Die meisten Strecken haben einen "Kopf" mit den Stammdaten und ein objects-Array mit den Streckenabschnitten. Du hast aber nur einen Streckenabschnitt, damit fällt das objects array weg - das hattest du aber vermutlich copy/paste gemacht und nicht sauber aufgeräumt.)

Bei sähe das nach dem fixen so aus: 

```suggestion
		{
			"group": 2,
			"maxSpeed": 120,
			"start": "KD",
			"end": "KSO",
			"twistingFactor": 0.2,
			"length": 19,
			"electrified": true
		},
```

---

Ich muss gestehen, dass ich es direkt in Github bearbeitet habe. Danke für die Hinweise, die werden mir in Zukunft weiterhelfen.